openpype/hosts/fusion/api/lib.py

```diff
@@ -302,10 +302,25 @@ def get_frame_path(path):
     return filename, padding, ext
 
 
-def get_current_comp():
-    """Hack to get current comp in this session"""
+def get_fusion():
+    """Get current Fusion instance"""
     fusion = getattr(sys.modules["__main__"], "fusion", None)
-    return fusion.CurrentComp if fusion else None
+    return fusion
+
+
+def get_current_comp():
+    """Get current comp in this session"""
+    fusion = get_fusion()
+    if fusion:
+        comp = fusion.CurrentComp
+        return comp
+
+
+def get_comp_filename():
```

Comment:
Why was this function created? The logic could stay in `current_file` as it's only place where is used.

Reply:
I used `get_comp_filename` [here](https://github.com/movalex/OpenPype/blob/807fd736fc7ce26b6e2553cfa9c1c53d7e13360f/openpype/hosts/fusion/api/workio.py#L28),  to maintain the .lib functions logic.

Reply:
That's my point. The function `get_comp_filename` is used only in `current_file` (which is requirement) so why to have 2 functions doing the same thing?

Reply:
Yeah, I see. Probably a misunderstanding here. I though the `lib` is a functions hub, and the `workio` is one of the modules that is using it. So while the func is used only once now, it may be used later by another modules. But I can revert that for sure.

Reply:
Well it's changing based on needs. Workio is/was categorized "lib hub" for workfiles. The lib.py is more likely uncategorized, or contains functions that are used on multiple places and there is no way how to put it to a better place to avoid cross imports. It matters on usage and host implementation (and developer). At this moment `get_comp_filename` is just a duplicate.